### DISCUSSION TITLE
Playing people in person on a shared single tablet

### DISCUSSION THREAD

**Discussion by CricketForJapan** [gbtami/pychess-variants#1554] [Ideas]
I can play other people in person on a shared single tablet using analysis.  However, it would be nice to add time controls and have the ability to resign. Then "Request Analysis".  I suppose username would be an issue, perhaps there could be an option to invite a guest in person, so the game would for my username would be "CricketForJapan" versus "CricketForJapan Guest".  All games obviously unrated.

**Comment by gbtami** [comment]
Allowing to edit PGN tags can be another option.

**Comment by CricketForJapan** [comment]
Shogi variants work quite well as the pieces are already reversed.  This is not the case for the chess variants, as black sees their pieces upside down.   I am not sure what you mean by PNG tags, is this allowed already or is this an option that needs to be introduced by the development team.  On branding, perhaps you could call any new options on pychess as "Street Chess".

**Comment by gbtami** [comment]
PNG tags (Event, Site, Date, White, Black, Result, Variant, FEN, SetUp) generated now below the board.

I think the Analysis page is not the best choice to re-use for this purpose though. Maybe adding a new play game mode, below the "PLAY WITH AI" button would be better. If you choose that, the new game dialog should be extended with a new input field "Opponent name".
